setup.py

```diff
@@ -14,7 +14,8 @@
     'six',
     'pytz',
     'pandas',
-    'jsonpath_ng'
+    'jsonpath_ng',
+    'dpanonymize'
```

Comment:
We should use `git+https://https://github.com/AMP-SCZ/dpanonymize.git` here. Otherwise, it will be burdensome to update PyPi content with every update of the former since the former is under active development.

Reply:
How about we leave it like this for now, and focus on running the test version for Pronet and Prescient, where I'll use `git clone lochness` and `git clone dpanonymize`?  We can come back to this later when our urgent data flow is taken care of and tested out.